sdk/confidentialledger/Azure.Storage.ConfidentialLedger/src/Generated/ConfidentialLedgerClient.cs

```diff
@@ -46,29 +44,94 @@ public ConfidentialLedgerClient(Uri ledgerUri, TokenCredential credential, Confi
             }
 
             options ??= new ConfidentialLedgerClientOptions();
-            Pipeline = HttpPipelineBuilder.Build(options, new BearerTokenAuthenticationPolicy(credential, AuthorizationScopes));
+            _clientDiagnostics = new ClientDiagnostics(options);
+            var authPolicy = new BearerTokenAuthenticationPolicy(credential, AuthorizationScopes);
+            Pipeline = HttpPipelineBuilder.Build(options, new HttpPipelinePolicy[] { authPolicy, new LowLevelCallbackPolicy() });
```

Comment:
Hmmm, I think it's OK for the preview, but we should not GA this. We will haver a mixture of generated methods and handcrafted methods on clients and it's not great that the generate methods have this policy in the pipeline.